### ISSUE TITLE
Different color for active unfocused workspace

### ISSUE THREAD

**Issue by setzer22** [denesb/xfce4-i3-workspaces-plugin#49]
When 2 or more screens are present, each screen is showing its *active* workspace, however, only the workspace with a focused window is considered *focused*.

I think there should be support for specifying three different colors for focused-active, unfocused-active and inactive, similarly to what i3bar has.

This would allow to see what is the visible workspace in a monitor even when it's not focused.

This should be a very small addition and I'm willing to work on it. Please let me know if the changes could be merged!

**Comment by denesb** [comment]
Yes, I would be happy to merge this.

**Comment by denesb** [comment]
Once this is merged as well I'll prepare a new release as we now have proper multi-monitor setup thanks to you.

**Comment by setzer22** [comment]
That's great news, glad to be of help! As you can see, I've just submitted a new PR (#50) with the changes,

Cheers!
